sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 12.0.0 (2021-06-08)
 
+### Other Changes and Improvements
+
+- Added and updated some samples.
+- Fixed a read consistency issue.
```

Comment:
Isn't this a bug fix? It doesn't matter at this point, but just asking to understand the scope of the issue.

Reply:
It's more of a behavior change than a bug fix.